build_release_package_json.js

```diff
@@ -7,8 +7,8 @@ try {
     const pkg = JSON.parse(origPackage);
     delete pkg.devDependencies;
     delete pkg.scripts;
-    pkg.main = 'index.js';
-    pkg.module = 'index.js';
+    pkg.main = 'src/index.js';
+    pkg.module = 'src/index.js';
```

Comment:
do you need to specify that the entry point has changed in package.json?

Reply:
Good catch